src/indexOf.js

```diff
@@ -1,7 +1,16 @@
-const indexOf = (array, value, [1]) =>{
-   {
-    return 
+const indexOf = (array, value, searchStart) =>{
+  let i = searchStart ? searchStart : 0
```

Comment:
Instead of using this turnery( ie `let i = searchStart ? searchStart : 0` ) you can do:
```
const indexOf = (array, value, searchStart=0) =>{
// all the smart things
}
```